carbon-design-system/carbon
# chore(project): update cypress and percy to latest

This PR updates cypress and percy to their latest versions

#### Changelog

**New**

**Changed**

- Update dependencies for percy and cypress
- Update `cypress.json` with new configuration options in v9

**Removed**

#### Testing / Reviewing

- Verify VRT tests work as expected
- Verify percy reports as expected
